Public/Src/Engine/UnitTests/Scheduler.IntegrationTest/FileAccessPolicyTests.cs

```diff
@@ -70,20 +70,12 @@ public void ValidateCachingUntrackedPathInput(bool declareDependency, bool input
 
             // Modify untracked input file /dir/nestedFile
             File.WriteAllText(ArtifactToString(nestedFile), "nestedFile");
-            if (declareDependency)
-            {
-                // Changes in declared dependencies always cause cache misses
-                RunScheduler().AssertCacheMiss(pip.PipId);
-            }
+

```

Comment:
With this PR, the semantics tested here clearly changed.  I think that's ok because I don't think anyone actually depends on this semantics.  I would just update the doc comment on this test method to say that content hashes of untracked paths are not added to the fingerprint.